ios/samples/KMSample1/KMSample1/AppDelegate.swift

```diff
@@ -15,8 +15,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
   func application(_ application: UIApplication,
                    didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]? = nil) -> Bool {
-    KeymanEngine.log.outputLevel = .debug
-    KeymanEngine.log.logAppDetails()
```

Comment:
Just... straight-up dropping the app-detail log?

Reply:
The details recorded in the console covers this pretty well for every log statement.